src/pages/team/index.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react";
+import TeamHero from "../../components/team/TeamHero";
+import TeamCard from "../../components/team/TeamCard";
+import { departmentsData } from "../../components/team/team-data";
+import FooterComp from "../../components/FooterComp";
+import { DepartmentsEnum } from "components/team/team-interfaces";
+import SubscriptionForm from "components/team/SubscriptionForm";
+
+const TeamPage = () => {
+	return (
+		<div className="team">
+			<TeamHero heroTitle="Our Team" />
+
+			<main className="container-fluid team-categories">
+				<div className="strokes">
+					<img src="/images/Team/Ellipse-right.png" alt="" id="ellipse-right" />
+					<img src="/images/Team/Ellipse-left.png" alt="" id="ellipse-left" />
+				</div>
+
+				<div className="desc">
+					<p id="rochester-p">Leadership</p>
+					<h4>Pastors & Elders</h4>
+				</div>
+				<section className="department container-sm">
+					{departmentsData
+						.filter((elem) => elem.department == DepartmentsEnum.LEADERSHIP)
+						.map((elem) => (
+							<TeamCard {...elem} key={elem.id} />
```

Comment:
Why do you have to spread the `elem` ? after destructuring it in `TeamCard` component

Reply:
Wanted to spread out elem and pass the data in as props of TeamCard component. 

please how should it be done instead?